src/PowerShellEditorServices.Protocol/LanguageServer/Completion.cs

```diff
@@ -86,6 +97,8 @@ public class CompletionItem
 
         public string InsertText { get; set; }
 
+        public InsertTextFormat InsertTextFormat { get; set; }
```

Comment:
Here:
```suggestion
        public InsertTextFormat InsertTextFormat { get; set; } = InsertTextFormat.PlainText;
```

Reply:
Although I thought enums defaulted to the first item in the enum. I could be wrong.

Reply:
It is 0, so it is unfortunate that this LSP starts at 1.

Reply:
That's so sad... oh well. I would prefer the property default value instead of the constructor, personally.